### ISSUE TITLE
Is there documentation?

### ISSUE THREAD

**Issue by BreninSul** [komputing/KEthereum#140]
Good afternoon, do you have any examples of how to use your library?

For example, getting events from blockchain, or decoding already existing event data in hex format into Kotlin objects?

I see that the code is written quite adequately, very strange that your library is not widespread, but there is absolutely no documentation =( Or am I missing something?

**Comment by ligi** [comment]
yea - documentation is quite lacking - PRs welcome!

**Comment by makarid** [comment]
Hello, i am interested to contribute on the project. Does the project still gets maintenance? Thanks

**Comment by ligi** [comment]
Yes - this project is still used and maintained - even though not that active currently.
